### DISCUSSION TITLE
error: Microsoft Visual C++ 14.0 or greater is required.

### DISCUSSION THREAD

**Discussion by ManuelMBaumann** [CERN/TIGRE#347] [Q&A]
Hi all,

I am using Windows and get the following error when running 'python setup.py install --user' out of my conda environment:

![image](https://user-images.githubusercontent.com/4333413/143026073-98047b43-4683-47a8-a9ed-a5770be5814b.png)

I then went on to install VS2022 but it seems to be located at a non-default location. Any hints on how to tell Python's setuptools where to find my VS path are very welcome!

**Comment by AnderBiguri** [comment]
Hi @ManuelMBaumann,

I have actually not tried with those, but MVS2017 should work as a tested alternative. 
The error may come from nvcc instead of just C++, which version of CUDA are you using?
